addons/udes_sale_stock/models/sale_order.py

```diff
@@ -334,10 +345,32 @@ def confirm_orders(self):
             # allow serialisation error to propagate.
             self.env.cr.commit()
             self.invalidate_cache()
+        if exception_data:
+            # Raise an exception including details of the exceptions that have
+            # suppressed, in case we want to expose this information somewhere.
+            collected_exceptions = '\n\n'.join(exception_data)
+            _logger.error(collected_exceptions)
+            raise CombinedException('At least one error occurred while confirming orders.',
+                                    collected_exceptions=collected_exceptions) from None
```

Comment:
from None?

Reply:
NVM I found why

Reply:
There's no benefit in chaining exceptions here, because this isn't a "real" exception.  At least, that's what I thought?